src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java

```diff
@@ -210,6 +220,11 @@ public String getColumnValue(final VariantContext variant) {
 
     AS_RAW_ReadPosRankSum {  // TODO -- maybe rely on 1/1 for call_GT
         public String getColumnValue(final VariantContext variant) {
+            // in the case where neither allele is reference, don't return a value
+            if (isGenotypeAllNonRef(variant.getGenotype(0))) {
+                return "";
+            }
+
```

Comment:
wow I didn't realize---we were entirely skipping annotations for 1/2 cases?

Reply:
no we weren't but we now are for the rank sum annotations as Laura recommended